src/main/resources/ca/corefacility/bioinformatics/irida/database/changesets/21.01/announcements_upgrade.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="announcement_upgrade" author="katherine">
+        <addColumn tableName="announcement">
+            <column name="title" type="varchar(255)" />
```

Comment:
This should probably also be `not null`.  It's a pain because we haven't set the titles yet with your python scripts.  An option would be to set a default value with the `value=something` tag here.  It could even be as simple as `Announcement title not found`.  At least that way we can set the not null constraint.

Reply:
Done.